src/searching.md

```diff
@@ -387,27 +388,26 @@ you need to tell Anki not to treat them specially.
   use `\\` instead of `\`.
 
 - `(` and `)`\
-  You can search for parentheses either by enclosing the full term in quotes,
-  and/or by using a backslash. That is, `"some(text)"`, `some\(text\)` and
-  `"some\(text\)"` are all equivalent, but `some(text)` is not.
+  You can search for parentheses by enclosing the entire term in quotes,
+   by using a backslash, or both at the same time. For example, `"(text)"`, `\(text\)` and
+  `"\(text\)"` are all equivalent searches, and searches for `(text)`.
```

Comment:
At the end, this should be:
`and search for `(text)`.`

But I don't agree with removing the more detailed examples that include "some" (unless they were incorrect in some way I'm not seeing?). I think they illustrate well that the entire term needs to be enclosed.

Reply:
```suggestion
  `"\(text\)"` are all equivalent searches, and search for `(text)`.
```
I believe the reader, if he has read other parts of this page/section will already understand what 'entire term' means. A simpler example is better here.